test/filters.test.js

```diff
@@ -64,4 +64,102 @@ describe('test/filters.test.js', function () {
       });
     });
   });
+  describe('BinaryComparator()', function () {
+    describe('write()', function () {
+      it ('should convert BinaryComparator to bytes', function () {
```

Comment:
See this https://github.com/alibaba/node-hbase-client/blob/master/test/client.test.js#L730